### DISCUSSION TITLE
ROM Document format unsupported

### DISCUSSION THREAD

**Discussion by alaskancodandcouponfraud** [haven1433/HexManiacAdvance#235] [Q&A]
Hi, I'm trying to load my ROM into HMA but get "The document's format is either unsupported or... contains binary characters."  It's just a standard GBA rom for Ruby. Am I doing something wrong? I have NET 6.0 installed and the latest version of Visual Studio. I'm relatively novice at code but can't for the life of me figure out how to get HMA to start with the game files as seen in all of the tutorials.

**Comment by haven1433** [comment]
> Visual Studio

You shouldn't need Visual Studio... did you download the source instead of the exe? The release has 3 downloads, try downloading the debug build.
